LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2018 Morgan Bartlett, Contract-Creativity (https://contract-creativity.com/)
```

Comment:
Copyright (c) 2018 Contract-Creativity (https://contract-creativity.com/): Morgan Bartlett

Reply:
We use to add the company first in other projects I use to collaborate